README.md

```diff
@@ -42,13 +42,15 @@ The contents of the file should be the following (see `.env.sample` for an examp
 
 ## Tests
 
-1. In order to run the tests of the project you need to have an instance of Redis running (as some of them test the integration with Redis).
+In order to run the test suite of the project:
+
+1. Have an instance of Redis running (as some of them test the integration with Redis).
 
 ```bash
 redis-server
```

Comment:
Shouldn't we encourage to use docker, as we do with the other projects?

Reply:
(I know you are just refactoring a little the document)

Reply:
Yes that is a good point. We have `docker-compose` also on the project so it shouldn't require a lot of configuration.

Reply:
I just checked and `docker-compose` is assigning a random port to `redis` because one is not set by default. I will update it to map the same port – that will make the instructions more clear.

Reply:
Opened it here: https://github.com/gnosis/safe-client-gateway/pull/765